### DISCUSSION TITLE
how do you change keybidns

### DISCUSSION THREAD

**Discussion by chaostheunderscore** [Leather128/LeathersFunkinEngine#18] [Q&A] [answered]
when i select how many keys i dont know how to change the keybinds for the keys i tried hitting enter, escape backspace, spacebar, ect what do i do?
![image](https://user-images.githubusercontent.com/89908006/139448937-4c9de2ef-2f7e-49aa-be0f-661481d7d4bd.png)

**Comment by Leather128** [comment, marked as answer]
just click on an arrow, and press the key you want to change it to

![image](https://user-images.githubusercontent.com/81660872/139510581-c8a55609-1c80-44e7-9ab1-dca06679a1a6.png)

**Reply by chaostheunderscore** [reply]
> just click on an arrow, and press the key you want to change it to
> 
> ![image](https://user-images.githubusercontent.com/81660872/139510581-c8a55609-1c80-44e7-9ab1-dca06679a1a6.png)

thanks!
